### ISSUE TITLE
Guitar bends playback not working

### ISSUE THREAD

**Issue by ijising** [musescore/MuseScore#24977]
### Issue type

Muse Sounds bug

### Description with steps to reproduce

some guitar bends dont play in playback

it can happen in the same bar where other bends work completely fine

any of the electric LP/electric SC can have this issue, i have not tested it with any of the acoustic soundfonts in muse guitars vol 1, the MS basic guitar soundfonts seem to work fine

copying and pasting the bar into another measure sometimes resolves the issue, but deleting the blank measure seems to reapply the bug onto the measure that took its place

things ive tried to resolve the issue but had no success: closing and reopening, copying and pasting the score on a separate blank score, pasting a working bend in place of the bugged one, changing the tuning of the target note in the bend, changing the tuning of the starting note in the bend, changing the note value, changing to another working soundfont and back, 

dont know how to reproduce the bug, it seems to happen at random. but i think this bug becomes more prominent the larger the file size (?)

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/b92ec043-d080-4cf1-8e52-013b42609dd7

ignore the second guitar, i was only testing out the other soundfonts but forgot to remove it when i recorded the clip

this bug also seems to pop up more later in the score, hence why im on measure 440. the bug can still appear in early bars

the score file in this clip is 155kb, if that matters at all

### What is the latest version of MuseScore Studio where this issue is present?

MS 4.2, when Muse Guitars vol 1 was added

### Regression

I was unable to check

### Operating system

Windows 11

### Additional context

i am very brand new to this platform so i apologize if ive breached a guideline

im also not very tech savvy so if there is a solution, please explain to me in laymans terms

any sort of help or response would be appreciated, thank you

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by FrancRos31** [comment]
I think this is a duplicate of #23604.

**Comment by ijising** [comment]
dwa

**Comment by ijising** [comment]
> I think this is a duplicate of #23604.

does that mean i have to delete this post or redirect my findings to that post?

**Comment by oktophonie** [comment]
We can close this one as a duplicate. If you wish to add any context to the other issue, feel free to do so!

**Comment by cbjeukendrup** [comment]
I'm not sure if it's a duplicate. https://github.com/musescore/MuseScore/issues/23604 is more about timing problems, whereas this issue is about the fact that bends sometimes randomly don't play back at all.

**Comment by cbjeukendrup** [comment]
Oops, thanks GitHub for now loading new activity while I had the tab open...

**Comment by ijising** [comment]
> We can close this one as a duplicate. If you wish to add any context to the other issue, feel free to do so!

it seems that my post was added in the comments, so i guess i have no need to copy everything over, ill continue to edit this post if i find anything new, thanks for telling me

**Comment by ijising** [comment]
> I'm not sure if it's a duplicate. #23604 is more about timing problems, whereas this issue is about the fact that bends sometimes randomly don't play back at all.

#23604 does briefly mention bends not playing near the very end, although it provides no details

im okay with this post being marked as duplicate if thats how things work around here
